libdnf.spec

```diff
@@ -35,6 +35,7 @@ BuildRequires:  gcc
 BuildRequires:  libsolv-devel >= %{libsolv_version}
 BuildRequires:  pkgconfig(librepo)
 BuildRequires:  pkgconfig(check)
+BuildRequires:  redhat-rpm-config
```

Comment:
Well, I was just trying to fix the problem with build and found out that missing file belongs to that package, didn't know about the background...

Should I remove it or keep it there temporary until bug will be fixed? Is there some workaround?